static/app/utils/theme.tsx

```diff
@@ -564,12 +564,14 @@ const generateButtonTheme = (colors: BaseColors, alias: Aliases) => ({
   },
 });
 
-const generateUtils = (colors: BaseColors) => ({
-  tooltipUnderline: {
-    textDecoration: `underline dotted ${colors.gray300}`,
+const generateUtils = (colors: BaseColors, aliases: Aliases) => ({
+  tooltipUnderline: (underlineColor: ColorOrAlias = 'gray300') => ({
+    textDecoration: `underline dotted ${
+      colors[underlineColor] ?? aliases[underlineColor]
+    }`,
     textDecorationThickness: '0.75px',
     textUnderlineOffset: '1.25px',
-  },
+  }),
```

Comment:
making this a function so we can change the underline color to better match the text, like here:
<img width="225" alt="Screen Shot 2022-04-21 at 11 42 07 AM" src="https://user-images.githubusercontent.com/44172267/164531271-c5ae6033-36b3-4f7b-ad0b-aac0280ebae4.png">